src/axom/core/tests/utils_config.cpp

```diff
@@ -59,9 +59,7 @@ TEST(core_config,config_libraries)
   libs.push_back("conduit");
 #endif
 
-#ifdef AXOM_USE_CXX11
-  libs.push_back("C++11");
-#endif
+libs.push_back("C++11");
```

Comment:
Should this be `AXOM_CXX_STD`  instead an explicit "C++11"?

Reply:
@kennyweiss Doesn't BLT and/or CMake have a variable for this? If so, any reason we should not use that?

Reply:
In this PR i mapped BLT_CXX_STD to AXOM_CXX_STD in the AxomConfig.hpp.  I left this on purpose because I was going to ask what the point of this test was.. because it just prints strings then returns true regardless.  Or am I missing something.

Reply:
Also I forgot, I have changed it to the AXOM_CXX_STD variable instead of being hard-coded.

Reply:
I don't understand the point of that test either....

I'll defer to your judgement about

Reply:
and the judgement of others about whether we alias axom variables to those in BLT/CMake or use them directly....

Reply:
I wrote that test as a quick way to check details about the configuration of Axom.
This was before we had the generated ``config.hpp`` or ``axom::about()``.

When @cyrush created the `about` example/exe, he copied most of this implementation into `about.cpp`, so I'd say that we no longer need the `utils_config` test, but should make sure that all details from here are present in `about.cpp` -- https://github.com/LLNL/axom/blob/develop/src/axom/core/utilities/About.cpp

Reply:
@rhornung67 -- (it is a bit complicated to describe, so I will try to be explicit).

Blt has a cmake variable `BLT_CXX_STD`, but does not generate a compiler `#define` for this, so we cannot use this information from the code.

In this PR, @white238 is removing axom's `AXOM_USE_CXX11` compiler define, but he added a new `AXOM_CXX_STD` compiler `#define` (as a string with the same value as the cmake variable `BLT_CXX_STD`), which we can use from the code, if we'd like.

Reply:
My original comment in this thread was that it would be better to use the new compiler define `AXOM_CXX_STD` rather than to hard code the string literal "C++11", since this will be more general. E.g. if a user configures axom with `C++14` or `C++17` support.